<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UyeIslemleri</class>
 <widget class="QDialog" name="UyeIslemleri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Üye İşlemleri</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="data_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>311</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_no">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_ad">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_soyad">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Üye No</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Üye Ad</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Üye Soyad</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_kayit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Yeni Kayıt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_guncelle">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>220</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Güncelle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sil">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Sil</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
